docs/qcqmc/high_level.ipynb

```diff
@@ -0,0 +1,286 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "679c5fd1",
+   "metadata": {},
+   "source": [
+    "# QCQMC\n",
+    "\n",
+    "This notebooks demonstrates the high-level code interface used to run the quantum part of https://arxiv.org/pdf/2106.16235.pdf.\n",
+    "\n",
+    "The code is organized into five steps, where the input of each step is specified by one or more `[Name]Params` dataclasses and the output of each step is an `attrs` dataclass named `[Name]Data` with a factory method `build_...`.  Each of the steps are demonstrated below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "16e16876",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import attrs\n",
+    "\n",
+    "def print_fields(x):\n",
+    "    \"\"\"Helper function to inspect returned objects' fields.\"\"\"\n",
+    "    for k, v in attrs.asdict(x).items():\n",
+    "        print(f'{k}: {type(v)}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "00d220ee",
+   "metadata": {},
+   "source": [
+    "## Hamiltonian\n",
+    "\n",
+    "The first step is to pick a molecule and generate or load its relevant physical properties, namely the Hamiltonian. Here we specify a 2-electron Fermi-Hubbard Hamiltonian in the sto3g basis. Integral data is stored in the data/ file in this repository.\n",
```

Comment:
What is the sto3g basis?  Is this a common terminology?  Can we link to a wikipedia or add a short description to explain it?

Reply:
It's common for chemists. I added a link to wikipedia.